pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -41,6 +41,13 @@ const (
 // +kubebuilder:validation:Enum=PrometheusProto;OpenMetricsText0.0.1;OpenMetricsText1.0.0;PrometheusText0.0.4
 type ScrapeProtocol string
 
+// RuntimeConfig configures the values for the process behavior.
+type RuntimeConfig struct {
+	// The Go garbage collection target percentage.
+	//+optional
+	GoGC *int `json:"gogc,omitempty"`
```

Comment:
API conventions recommend int32 or int64 for clarity. Negative values are possible (equivalent to GOGC=off) but we can restrict to -1.
 
```suggestion
	// +kubebuilder:validation:Minimum=-1
	GoGC *int32 `json:"gogc,omitempty"`
```

Reply:
Since it is a percentage, I think it also makes sense to have a Maximum value of 100
`` // +kubebuilder:validation:Maximum=100``

Reply:
Right, did not know that